src/mlpack/methods/ann/layer/base_layer.hpp

```diff
@@ -264,6 +266,17 @@ template <
 using ElishFunctionLayer = BaseLayer<
     ActivationFunction, InputDataType, OutputDataType>;
 
+/**
+ * Standard ELiSH-Layer using the ELiSH activation function.
+ */
+template <
+    class ActivationFunction = GaussianFunction,
```

Comment:
can you add a template parameter like this in the base layer?

Reply:
Done